src/core/jsdoc-tsd-parser.ts

```diff
@@ -350,8 +378,87 @@ export class JSDocTsdParser {
 								}
 								break;
 
+							case "interface":
+								let objectTypeMember = parsedItem as dom.ObjectTypeMember;
+
+								switch ((objectTypeMember as any).kind) {
+									case "function":
+										let functionDeclaration: any = objectTypeMember as any;
+										objectTypeMember = {
+											kind: "method",
+											name: functionDeclaration.name,
+											parameters: functionDeclaration.parameters,
+											returnType: functionDeclaration.returnType,
+											typeParameters: functionDeclaration.typeParameters
+										};
+
+										objectTypeMember.jsDocComment = functionDeclaration.jsDocComment;
+										break;
+
+									case "property":
+										// ok, nothing to change
+										break;
+
+									case "enum":
+										let propertyDeclaration: any = objectTypeMember as any;
+										objectTypeMember = {
+											kind: "property",
+											name: propertyDeclaration.name,
+											type: "number"
+										};
+										break;
+
+									default:
+										console.warn("Member " + (objectTypeMember as any).name + " with unexpected kind '" + (objectTypeMember as any).kind + "' in module " + parentItem.name);
+										break;
+								}
+
+								(parentItem as dom.InterfaceDeclaration).members.push(objectTypeMember);
+								break;
+
+							case "module":
+								let moduleMember = parsedItem as dom.ModuleMember;
+
+								switch ((moduleMember as any).kind) {
+									case "member":

```

Comment:
You make every module member to a constant member which is not correct. A non-constant module member would be from type "var". This is currently not supported by the dts-dom module so we have to create a pull request to make this type of module members available.
For now you can either remove these cases or add the ability to parse constants. Then you have to fix this cases that only constants will be exported to constants